src/main/java/uk/gov/hmcts/divorce/caseworker/event/page/AmendCase.java

```diff
@@ -14,6 +17,65 @@ public void addTo(PageBuilder pageBuilder) {
             .complex(CaseData::getApplication)
                 .complex(Application::getMarriageDetails)
                     .optional(MarriageDetails::getDate)
+                .done()
+            .done()
+            .readonly(CaseData::getApplicationType,
```

Comment:
this should be readonly no summary ? otherwise will be shown on summary page ?

Reply:
Will add configuration to enable summary page. Verified it with Divorce 1 configuration.